packages/documentation-v7/src/stories/components/choice-card/radiobutton-card.stories.ts

```diff
@@ -0,0 +1,20 @@
+import { BADGE } from '../../../../.storybook/constants';
+import { choiceCardMeta, choiceCardDefault, choiceCardGroup } from './choice-control';
+
+export default {
+  ...choiceCardMeta,
+  title: 'Components/Radio Button Card',
+  parameters: {
+    badges: [BADGE.NEEDS_REVISION],
+  },
+};
+
+export const Default = {
+  render: choiceCardDefault,
+  args: { ...choiceCardMeta.args, type: 'radio' },
+};
+
+export const Group = {
+  render: choiceCardGroup,
```

Comment:
```suggestion
  ...choiceCardGroup,
```

Reply:
choiceCardDefault is a simple function, not a story. This would not work.